apps/storefront/src/gatsby-theme-docz/components/MainContainer/index.js

```diff
@@ -121,75 +129,91 @@ export const MainContainer = ({ children, doc, ...rest }) => {
   const isContentPage = type !== 'landingPage'
   const isPublished = (mode || '').toLowerCase() === 'publish'
   const current = useCurrentDoc()
+  const isPrivate =
+    mode.toLowerCase() === 'draft' && process.env.GATSBY_STAGE === 'prod'
 
   return (
     <main {...rest} id="main">
       {isContentPage ? (
-        <>
-          <ContentHeader withTabs={withTabs}>
-            <H1>
-              {title}
-              {!isPublished && (
-                <span className={`ModeBadge ModeBadge--${mode}`}>
-                  {mode && `(${mode})`}
-                </span>
-              )}
-            </H1>
-            {tabs && (
-              <nav aria-label="tabbed content naviagtion">
-                <Tabs>
-                  {tabs.map((tab, index) => {
-                    const routeSegment = tab.toLowerCase().replace(/\s/g, '-')
-                    const firstTwoRouteSegments = /^\/([a-z-]+\/?){2}/
-                    const categoryRoute = route.match(firstTwoRouteSegments)[0]
+        isPrivate ? (
+          <>
+            <Helmet>
+              <meta name="robots" content="noindex" />
+            </Helmet>
+            <ContentHeader>
+              <H1>Placeholder</H1>
```

Comment:
Show the text “placeholder” instead of actual header to prevent showing up in search results